components/CustomTable.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import {
+  TableContainer,
+  Table,
+  TableHead,
+  TableRow,
+  TableCell,
+  TableBody,
+} from '@mui/material';
+
+const CustomTable = ({ headerRow, data }) => {
+  return (
```

Comment:
no need to write return, if there's nothing in the function body just do () => (<TableContainer>... </TableContainer>)